test/testenv/buildx.go

```diff
@@ -292,6 +307,16 @@ func (b *BuildxEnv) RunTest(ctx context.Context, t *testing.T, f TestFunc, opts
 	withProjectRoot(t, &so)
 	withResolveLocal(&so)
 
+	withSecrets(&so,
```

Comment:
Seems like secrets should be a `TestRunnerOpt`

Reply:
Also doesn't seem like we're actually using this in any tests?

Reply:
It got added to the main `RunTest`. I'll move it to the test proper as a `TestRunnerOpt`.

Reply:
Nevermind. This was before I got rid of the integration tests.